tests/headers/file_handling/cpp_comments.h

```diff
@@ -1,6 +1,6 @@
 // This is file is used to test the removal of C++ style comments
 
-# define MACRO 1 // inline comment
+# define MACRO 2 // inline comment
```

Comment:
During development of f3ed1ba1c2671391df94d31940f048828bd725cb I used cpp_comments.h for test_process_multiple_files. Thus I needed different values for the ```MACRO``` macro to check if the last parsed file is overwriting the macro definitions of the earlier files.

But then I reworked this by providing extra files for this test, instead of misusing ```cpp_comments.h``` without undoing this (then unnecessary change)...